README.md

```diff
@@ -680,8 +681,19 @@ No equivalent of the ImageStream SCM (SCM steps are a special extension point in
 That step was introduced prior to the introduction of the OpenShift Pipeline Build Strategy.
 
 With the advent of OpenShift Pipeline Build Strategy, incorporating your pipeline into such a BuildConfig along with the use of an Image Change Trigger is the better choice for triggering pipeline jobs from changes to ImageStreams in OpenShift.
-   
 
+### Tagging images across namespaces
```

Comment:
@itwasonlyabug Thank you for the PR :)

Can you also add a little but if verbose description as to what you are doing in this example ? 


/assign waveywaves

Reply:
Hi! Do you mean as in the [description under the heading here](https://github.com/openshift/jenkins-client-plugin#sticking-with-the-defaults) or with more comments?

I want to keep it consistent with the rest, but the style varies a little :)

I could also add other examples for tagging images, but I'm not sure if the use-case would be helpful.

Reply:
Description under the header should be good.

Reply:
Looks good.